src/nvim/option.c

```diff
@@ -1442,9 +1442,8 @@ int do_set(char *arg, int opt_flags)
       } else {
         char *startarg = arg;             // remember for error message
         char *errmsg = NULL;
-        char errbuf[80];
 
-        do_set_option(opt_flags, &arg, &did_show, errbuf, sizeof(errbuf), &errmsg);
+        do_set_option(opt_flags, &arg, &did_show, IObuff, IOSIZE, &errmsg);
```

Comment:
What is the point of this? Why create more potential conflict between functions?

Reply:
You're right, I'll undo this.